cypress/fixtures/seeds/granulesFixture.json

```diff
@@ -52,10 +52,12 @@
         }
       ],
       "collectionId": "MOD09GQ___006",
+      "producerGranuleId": "MOD09GQ.A1530852.CljGDp.006.2163412421938",
```

Comment:
I wonder if there's any value in having one or two of these fixtures with a a producerGranuleId != granuleId. They will be the same in most cases probably but uniquified granules will have different values.

Doesn't really matter but it might make it more realistic?

Reply:
https://bugs.earthdata.nasa.gov/browse/CUMULUS-4086 This should take care of that.

Reply:
I can make a few uniquified granule ids.